### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by kwsorensen**

### CODE REVIEW

**Comment by kwsorensen** [comment]
#7

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
```
Comment (aro5000) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: These are probably less relevant to us with a CLI tool - maybe something about K8s version would be better?

Or maybe take an example from helm? https://github.com/helm/helm/blob/main/.github/issue_template.md

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
```
Comment (aro5000) [inline .github/ISSUE_TEMPLATE/feature_request.md:8]: Does this table automatically link in GitHub? If not, I feel like we should just omit this - unless we give guidance to contributors on what to put for `labels` and `assignees` or would the expectation be that maintainers fill that out?

Comment (kwsorensen) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Looks like my edits didn't save 👎

**Review by aro5000 — APPROVED** [review]

**Pull request merged**